Lara-JS/src-api/lara/util/DataStore.ts

```diff
@@ -0,0 +1,270 @@
+import Io from "../Io.js";
+import { info } from "../core/LaraCore.js";
+import JavaTypes, { JavaClasses } from "./JavaTypes.js";
+
+/**
+ * Interface with Java class DataStore, used for storing arbitrary information.
+ */
+
+export default class DataStore {
+  javaDataStoreInstance!: JavaClasses.DataStore;
+  definition: any;
```

Comment:
This is a StoreDefinition (Java Interface), in case you want to add this information